src/lexer/token_string_utils.c

```diff
@@ -0,0 +1,74 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   token_string_utils.c                               :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: lporoshi <lporoshi@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/12/18 13:11:42 by lporoshi          #+#    #+#             */
+/*   Updated: 2023/12/19 14:41:05 by lporoshi         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "libft.h"
+#include "lexer.h"
+
+static int	parse_two_sym_word(char *line)
+{
+	if (line[0] == line[1])
+		return (2);
+	return (1);
+}
+
+static int	parse_quotes(char *line, const char quote)
+{
+	int	i;
+
+	i = 1;
+	while (line[i] != '\0' && line[i] != quote)
```

Comment:
do we need to handle escape characters (like \\")?

Reply:
The subject i ambiguous about this, but no, we don't need to. We can treat backslashes the same we treat normal characters.
Here's an official answer to this quesion.
<img width="1082" alt="Screen Shot 2023-12-21 at 3 56 41 PM" src="https://github.com/taaae/minishell/assets/80582995/f42d10e1-fa40-406b-9fc0-729612bc6e0a">